File: app/src/main/java/com/gmail/fattazzo/formula1world/ergast/json/objects/Season.java
package com.gmail.fattazzo.formula1world.ergast.json.objects;

/**
 * @author fattazzo
 *         <p/>
 *         date: 06/06/17
 */
public class Season {
    private int season;
    private String url;

    public Season(String season, String url) {
        this.season = Integer.valueOf(season);
        this.url = url;
    }

    @Override
    public String toString() {
        return "Season{" +
                "season=" + season +
                ", url='" + url + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/com/gmail/fattazzo/formula1world/ergast/json/objects/Season.kt
package com.gmail.fattazzo.formula1world.ergast.json.objects

/**
 * @author fattazzo
 *
 *
 * date: 06/06/17
 */
class Season(season: String, private val url: String) {
    private val season: Int

    init {
        this.season = Integer.valueOf(season)!!
    }

    override fun toString(): String {
        return "Season{" +
                "season=" + season +
                ", url='" + url + '\'' +
                '}'
    }
}
